src/picongpu/include/plugins/kernel/CopySpecies.kernel

```diff
@@ -33,8 +33,104 @@ namespace picongpu
 
 using namespace PMacc;
 
+/** copy particle to single frame
+ *
+ * copy particle data which are stored in linked list and superCell hierarchy
+ * to a single frame.
+ */
+struct CopySpeciesToGlobal
```

Comment:
this naming might be confusing ("global" is not "global memory").

proposed renaming: `CopySpeciesToArray` or `TransformSpeciesToArray`